src/app/content/components/SearchResultsSidebar.tsx

```diff
@@ -0,0 +1,278 @@
+import { SearchResult } from '@openstax/open-search-client/dist/models/SearchResult';
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { connect } from 'react-redux';
+import styled, { css } from 'styled-components/macro';
+import { Search } from 'styled-icons/fa-solid/Search';
+import { Times } from 'styled-icons/fa-solid/Times/Times';
+import { Details, iconSize } from '../../components/Details';
+import { navDesktopHeight, navMobileHeight } from '../../components/NavBar';
+import { labelStyle, textRegularLineHeight, textRegularStyle } from '../../components/Typography';
+import theme from '../../theme';
+import { AppState } from '../../types';
+import * as selectSearch from '../search/selectors';
+import {
+  bookBannerDesktopMiniHeight,
+  bookBannerMobileMiniHeight,
+  mobileSearchContainerMargin,
+  searchResultsBarDesktopWidth,
+  searchResultsBarMobileWidth,
+  sidebarDesktopWidth,
+  sidebarMobileWidth,
+  toolbarDesktopHeight,
+  toolbarMobileHeight,
+  toolbarSearchInputMobileHeight,
+} from './constants';
+import { CollapseIcon, ExpandIcon, Summary, SummaryTitle, SummaryWrapper } from './Sidebar/styled';
+import { toolbarIconStyles } from './Toolbar';
+
+const searchResultsBarVariables = {
+    backgroundColor: '#f1f1f1',
+    mainPaddingDesktop: 3,
+    mainPaddingMobile: 2,
+};
+
+// tslint:disable-next-line:variable-name
+const SearchIconInsideBar = styled(Search)`
+  ${toolbarIconStyles}
+  color: ${theme.color.primary.gray.darker};
+  margin-right: 0.7rem;
+  margin-left: 3rem;
+`;
+
+// tslint:disable-next-line:variable-name
+const CloseIcon = styled(Times)`
+  ${toolbarIconStyles}
+  color: ${theme.color.primary.gray.lighter};
+  margin-right: 1.4rem;
+`;
+
+// tslint:disable-next-line:variable-name
+const SearchResultsBar = styled.div`
+  top: calc(${bookBannerDesktopMiniHeight}rem + ${toolbarDesktopHeight}rem);
+  overflow-y: auto;
+  height: calc(100vh - ${navDesktopHeight + bookBannerDesktopMiniHeight}rem);
+  transition: transform 300ms ease-in-out,box-shadow 300ms ease-in-out,background-color 300ms ease-in-out;
+  background-color: #fafafa;
+  z-index: 4;
+  margin-left: calc(-50vw - ${sidebarDesktopWidth}rem - ${searchResultsBarVariables.mainPaddingDesktop}rem);
+  padding-left: 50vw;
+  width: calc(50vw + ${searchResultsBarDesktopWidth}rem);
+  min-width: calc(50vw + ${searchResultsBarDesktopWidth}rem);
+  box-shadow: 0.2rem 0 0.2rem 0 rgba(0,0,0,0.1);
+  display: flex;
+  flex-direction: column;
+  position: sticky;
+  background: ${searchResultsBarVariables.backgroundColor};
+
+  &:not([open]) {
+    display: none;
+  }
+
+  &[open] {
+    display: block;
+  }
+
+  ${theme.breakpoints.mobile(css`
+    width: calc(50vw + ${searchResultsBarMobileWidth}vw);
+    min-width: calc(50vw + ${searchResultsBarMobileWidth}vw);
+    top: ${bookBannerMobileMiniHeight + toolbarMobileHeight
```

Comment:
we're gonna have to give this access to whether the search input is open on mobile and compensate so the top position is right whether its open or closed

probably do the same in the toc sidebar

thats sort of a pain...